Tools/Scripts/webkitpy/layout_tests/controllers/layout_test_finder_legacy.py

```diff
@@ -164,7 +169,8 @@ def _expand_variants(self, files):
                         variants.append(variant)
                 if variants:
                     for variant in variants:
-                        expanded.append(f + variant)
+                        if not passed_variant or variant.startswith(passed_variant):
+                            expanded.append(f + variant)
```

Comment:
with the above suggestion:
```suggestion
                        if not passed_variant or variant.startswith(variant_separator + passed_variant):
                            expanded.append(f + variant)
```